src/components/inputs/rich-text-input/rich-text-input.visualspec.js

```diff
@@ -336,5 +336,33 @@ describe('RichTextInput', () => {
 
     numOfTags = await getNumberOfTags('li');
     expect(numOfTags).toEqual(2);
+
+    // remove all the text
+    await selectAllText(input);
+    await input.press('Backspace');
+
```

Comment:
this test basically shows that we can externally control the value, and then still use the input after